src/lib.rs

```diff
@@ -8,6 +8,11 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
+#[macro_use]
```

Comment:
Oh, interesting, so you get global logging without an explicit reference to a logger object.  Does this work across crates?  I.e., if we added `info!` logging to the `query` crate, would this automatically pick up the logger you configure in `main.rs` below?